### PR TITLE
biobloomtools: import from brewsci/science

### PR DESCRIPTION
**Pull request by gejessicama**

### CODE REVIEW

**Comment by sjackman** [comment]
Depends on `zlib` on Linux.
```
==> brew linkage --test brewsci/bio/biobloomtools
Missing libraries:
  libz.so.1
```
https://travis-ci.org/brewsci/homebrew-bio/builds/363306334#L2748-L2750

**Comment by sjackman** [comment]
The `--help` output is printed to standard error rather than standard output.
See https://circleci.com/gh/brewsci/homebrew-bio/1898

File: Formula/biobloomtools.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  head do
+    url "https://github.com/bcgsc/biobloom.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  fails_with :clang # needs openmp
+
+  depends_on "boost" => :build
+  depends_on "gcc" if OS.mac? # for openmp
```
Comment (sjackman) [inline Formula/biobloomtools.rb]: Change this line to…
```ruby
if OS.mac?
  depends_on "gcc" # for openmp
else
  depends_on "zlib"
end
```

File: Formula/biobloomtools.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "--disable-debug",
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "Usage", shell_output("#{bin}/biobloommaker --help")
+    assert_match "Usage", shell_output("#{bin}/biobloomcategorizer --help")
```
Comment (sjackman) [inline Formula/biobloomtools.rb]: Change both `--help` to `--help 2>&1` to redirect standard error to standard output.

**Comment by sjackman** [comment]
Thanks for your first contribution to Brewsci/bio, Jessica! Two minor fixes are needed. (see above)

**Comment by gejessicama** [comment]
Hi Shaun,

Would you want these also reflected in the DIDA migration?

**Comment by sjackman** [comment]
DIDA may be different. Submit it without these changes and see what happens.
You can run
```sh
brew linkage dida
```
locally to see whether it depends on `zlib`.

**Comment by gejessicama** [comment]
Looks like there were some problems with my whitespace. I tried to fix it, not sure if I did.

**Comment by sjackman** [comment]
```
configure: error: Requires the Boost C++ libraries
```
You accidentally deleted
```ruby
depends_on "boost" => :build
```
or misinterpreted my comment, in which case sorry for the confusion!

**Comment by sjackman** [comment]
🍏 Success!

**Comment by sjackman** [comment]
Merged. Thank you for your first contribution to Brewsci/bio, Jessica! 🏅

**Pull request closed without merging**